Documentation/CSSProperties.md

```diff
@@ -0,0 +1,53 @@
+# Adding or Modifying a CSS Property
+
+There are several different places you need to make changes, in order to add a CSS property, or later modify it.
```

Comment:
"in order to add a CSS property" shouldn't have commas around it.
I'd also rewrite "or later modify it" to "or to modify it later"